deployments/get/objects.go

```diff
@@ -413,6 +414,8 @@ func GenerateDaemonSet(env, namespace string) *appsv1.DaemonSet {
 							Image:           "kubearmor/kubearmor:stable",
 							ImagePullPolicy: "Always",
 							SecurityContext: &corev1.SecurityContext{
+								RunAsUser:  ptr.To(int64(1000)),
+								RunAsGroup: ptr.To(int64(1000)),
```

Comment:
Let's avoid this and change the Dockerfile instead to use non root user

Reply:
Dockerfile already have configuration to use non-root user